FirebaseDatabase.podspec

```diff
@@ -32,6 +32,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.libraries = ['c++', 'icucore']
   s.frameworks = ['CFNetwork', 'Security', 'SystemConfiguration']
   s.dependency 'leveldb-library', '~> 1.18'
+  s.dependency 'FirebaseAuthInterop', '~> 1.0'
   s.dependency 'FirebaseCore', '~> 5.0'
```

Comment:
`FirebaseCore` also needs to go to `5.1` and above - we missed this before in the other PRs. `5.1` is where the component system was introduced.

Reply:
Done.